types/chrome/index.d.ts

```diff
@@ -1845,6 +1850,14 @@ declare namespace chrome.cookies {
         sameSite: SameSiteStatus;
     }
 
+    /** Represents a partitioned cookie's partition key. */
+    export interface CookiePartitionKey {
+        /** Indicates if the cookie was set in a cross-cross site context. This prevents a top-level site embedded in a cross-site context from accessing cookies set by the top-level site in a same-site context. */
+        hasCrossSiteAncestor?: boolean;
```

Comment:
The docs say this property is pending (not yet in a stable release). IMO, this shouldn't be in the types if it isn't in the actual API yet.

Reply:
done